docs/rfcs/blocks-in-widget-areas.md

```diff
@@ -0,0 +1,520 @@
+- [WP35669][] discusses storing widgets in a custom post type.
+
+[GB14182]: https://github.com/WordPress/gutenberg/issues/14182
+[GB13204]: https://github.com/WordPress/gutenberg/issues/13204
+[GB13205]: https://github.com/WordPress/gutenberg/issues/13205
+[GB13511]: https://github.com/WordPress/gutenberg/issues/13511
+[WP41683]: https://core.trac.wordpress.org/ticket/41683
+[WP33507]: https://core.trac.wordpress.org/ticket/33507
+[WP35574]: https://core.trac.wordpress.org/ticket/35574
+[WP35669]: https://core.trac.wordpress.org/ticket/35669
+
+## Frontend
+
+A new `@wordpress/edit-widgets` package contains the UI for the `wp-admin/widgets.php` screen. This is analogous to how `@wordpress/edit-post` contains the UI for `wp-admin/post-new.php`. 
+
+Both WP Admin and the Customizer use the existing [`BlockEditor`][block-editor] component, which provides a generic UI for working with blocks, to display an interface for working with blocks in widget-editing areas.
+
+A new REST API, described in the next section, allows fetching and updating the blocks in a widget area. `@wordpress/edit-widgets` uses this API to fetch blocks which are passed to `BlockEditor`, and to update blocks when `BlockEditor` indicates that changes have been made by the user.
+
+```js
+function WidgetArea( { id } ) {
+	return (
+		<BlockEditor
+			// Illustrative only: apiGet() is not a real function. In production,
+			// @wordpress/core-data would be used to interface with the REST API.
+			blocks={ apiGet( `/wp/v2/widget-areas/${ id }` ) }
+			onChange={ ( blocks ) => {
+				// Illustrative only: apiPut() is not a real function. In production,
+				// @wordpress/core-data would be used to interface with the REST API.
+				apiPut( `/wp/v2/widget-areas/${ id }`, blocks );
+			} }
+		/>
+	);
+}
+```
+
+[block-editor]: https://github.com/WordPress/gutenberg/tree/master/packages/block-editor
+
+## REST API
+
+Fetching and updating the blocks in a widget area is done via new REST API endpoints.
+
+### The widget area resource
+
+The REST API is built around a single resource object: the widget area.
+
+```json
+{
+	"id": "footer",
+	"name": "Footer",
+	"description": "",
+	"content": {
+		"raw": "
+			<!-- wp:paragraph -->
+			<p>Hello there!</p>
+			<!-- /wp:paragraph -->
+
+			<!-- wp:legacy-widget {\"identifier\":\"search-2\",\"instance\":{\"title\":\"\"}} /-->
+		",
+		"rendered": "
+			<p>Hello there!</p>
+			<form role=\"search\" method=\"get\" class=\"search-form\" action=\"http://wordpress.test/\">
+				...
+			</form>
+		",
+	},
+}
+```
+
+#### `id`
+
+- Type: `string`
+- Access: Read-only
+
+```json
+{ "id": "footer" }
+```
+
+The unique identifier for the widget area. This is set by the theme using [`register_sidebar()`][register_sidebar]. 
+
+`"wp_inactive"` is a special identifier used for blocks that belong to widget areas that are no longer registered.
+
+[register_sidebar]: https://developer.wordpress.org/reference/functions/register_sidebar/
+
+#### `name`
```

Comment:
What field does this map to?

Reply:
It maps to the title given to [`register_sidebar()`](https://developer.wordpress.org/reference/functions/register_sidebar/) by the theme.